roles/openshift_autoheal/tasks/upgrade.yml

```diff
@@ -0,0 +1 @@
+# This is empty on purpose as there is nothing to upgrade.
```

Comment:
We need to support going from 3.10.0 to 3.10.1 with this task file.  Also, this should support upgrades generally, 3.10 to 3.11.

We require some form of an upgrade play before the role merges.

Reply:
@michaelgugino I think you're wrong here. I don't think `upgrade.yml` should exist *at all* for this role. Like a lot of other roles in openshift-ansible, the proper way to upgrade autoheal would just be changing the version number and then running the `install` tasks again.

A separate upgrade task file should only be needed if there's something special needed to be done on upgrades (like configuration migration). This is not the case here.

Reply:
Yes, it's definitely ideal if upgrading is as simple as calling the install procedures over again. Just call that from your upgrade playbook, as it is now you have an upgrade playbook that's a NOOP. Then if something in the future does require specific logic for upgrades people continue calling the same playbook as before to upgrade and we get that custom migration.